main.go

```diff
@@ -94,51 +120,76 @@ func getJoke(w http.ResponseWriter, r *http.Request) {
 				result = append(result, item)
 			}
 		}
-		t.ExecuteTemplate(w, "findjoke", result)
+		err := t.ExecuteTemplate(w, "findjoke", result)
+		if err != nil {
+			log.Fatal(err)
+		}
 	}
 
 	//Searching by ID
 	if id != "" {
 		for _, item := range jokes {
 			if item.ID == id {
 				result = append(result, item)
-				t.ExecuteTemplate(w, "findjoke", result)
+				err := t.ExecuteTemplate(w, "findjoke", result)
+				if err != nil {
+					log.Fatal(err)
+				}
 			}
 		}
 	}
 }
 
 func getRandomJokes(w http.ResponseWriter, r *http.Request) {
-	t, err := template.ParseFiles("templates/random.html", "templates/header.html", "templates/footer.html")
-	if err != nil {
-		fmt.Fprintf(w, err.Error())
-	}
 	s := rand.NewSource(time.Now().UnixNano())
 	rnd := rand.New(s)
 
+	skip, err := strconv.Atoi(r.URL.Query().Get("skip"))
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	seed, err := strconv.Atoi(r.URL.Query().Get("seed"))
+	if err != nil {
+		log.Fatal(err)
+	}
+
 	//Filling an array with random elements
 	var rndjokes [100]Joke
 	for i := 0; i < 100; i++ {
 		rndjokes[i] = jokes[rnd.Intn(len(jokes))]
 	}
 
-	t.ExecuteTemplate(w, "random", rndjokes)
+	err = t.ExecuteTemplate(w, "random", rndjokes[skip:skip+seed])
+	if err != nil {
+		log.Fatal(err)
+	}
 }
 
 func getFunniestJokes(w http.ResponseWriter, r *http.Request) {
-	t, err := template.ParseFiles("templates/funniest.html", "templates/header.html", "templates/footer.html")
+	//Sorting an array by score
+
+	skip, err := strconv.Atoi(r.URL.Query().Get("skip"))
```

Comment:
I noticed that skip and seed used mupltiple times. Maybe it worse to combine this logic into single function `getPaginationParams` or `getSeedAndSkip` ?